backend/lib/plugins/socket.js

```diff
@@ -461,6 +461,8 @@ function onSocketConnect(socket) {
     const { userId } = socket;
     const [errPost, post] = await this.to(Post.findById(data.postId));
     if (errPost || !post) return;
+    post.shares += 1;
+    await post.save();
```

Comment:
maybe change to use `findByIdAndUpdate` with `$inc` clause, which will be atomic
also add error handling